### ISSUE TITLE
Front-End: API

### ISSUE THREAD

**Issue by aditore** [aditore/chatApp#31]
build API in utils/API.js call backend API

**Comment by ArshErgon** [comment]
hey @aditore so here we want to create a `API.js` inside `utils` right? I'm quite new in open source, but I think I can do this.
How will I be running this locally, there's nothing in `ReadMe.md`

**Comment by aditore** [comment]
Hey!

Yes sorry the readme hasn’t been made because it’s a new project, sorry about that

We need to call the api in the utils folder, but first we need to set up the controller in the backend to allow the front end calls to do something

If you clone the repo and npm install you’ll get all the packages that you need, then you need to source the schema in MySQL  to add the database to MySQL. Then just run npm run develop from the root and it should show everything we have so far 👍🏽

**Comment by aditore** [comment]
The api can still be built out first, as long as the calls are all correct 👍🏽

**Comment by aditore** [comment]
Also make sure to create a .env file with your sql username, password and the name of the database (which is chat_db)

**Comment by ArshErgon** [comment]
@aditore this looks hard, but I will find a way, thanks

**Comment by aditore** [comment]
It's all about getting it done and learning something from it ! Let me know if you have anymore questions !

**Comment by ArshErgon** [comment]
> then you need to source the schema in MySQL to add the database to MySQL.

do I have to run this command locally on my local MySQL?

**Comment by aditore** [comment]
Yes, I believe so., sequelize might do it for you but am unsure. 

So I sourced it locally. There isn't an online server that holds our database yet. 

So for best development practicing, creating database on local sql is the way to go!
